ui/app/serializers/config-ui/message.js

```diff
@@ -3,22 +3,37 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import { encodeString } from 'core/utils/b64';
+import { decodeString, encodeString } from 'core/utils/b64';
 import ApplicationSerializer from '../application';
 
 export default class MessageSerializer extends ApplicationSerializer {
   primaryKey = 'id';
 
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    if (requestType === 'queryRecord') {
+      const transformed = {
+        ...payload.data,
+        message: decodeString(payload.data.message),
+        link_title: payload.data.link.title,
+        link_href: payload.data.link.href,
+      };
+      delete transformed.link;
```

Comment:
instead of deleting this here, does it work if you do something above about attributes you do not want to serialize? I ask because it makes it easier to debug later (I'm noticing this in ember data upgrade things) if all attributes removed are listed at the top of the serializer.

Reply:
```
 attrs = {
    link: { serialize: false },
  };
```

Reply:
I'll address the link attribute feedback in a different PR :) Ticket to address this feedback is VAULT-22908